src/rdb_protocol/datum_stream.hpp

```diff
@@ -404,6 +405,7 @@ class range_datum_stream_t : public eager_datum_stream_t {
         return false;
     }
     virtual bool is_exhausted() const;
+    virtual bool is_in_range() const;
```

Comment:
This doesn't need to be `virtual`. The other functions are virtual only because they're overwriting functions declared in `eager_datum_stream_t`.